src/sections/Workshops.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from 'react';
+import styled from 'styled-components'
+import nugget from '../../public/assets/images/nugget.svg'
+import apis1 from '../../public/assets/images/apis1.svg'
+import apis2 from '../../public/assets/images/apis2.svg'
+import hackathons1 from '../../public/assets/images/hackathons1.svg'
+import hackathons2 from '../../public/assets/images/hackathons2.svg'
+import figma1 from '../../public/assets/images/figma1.svg'
+import figma2 from '../../public/assets/images/figma2.svg'
+import internships1 from '../../public/assets/images/internships1.svg'
+import internships2 from '../../public/assets/images/internships2.svg'
+import react1 from '../../public/assets/images/react1.svg'
+import react2 from '../../public/assets/images/react2.svg'
+import version1 from '../../public/assets/images/version1.svg'
+import version2 from '../../public/assets/images/version2.svg'
+import webdev1 from '../../public/assets/images/webdev1.svg'
+import webdev2 from '../../public/assets/images/webdev2.svg'
+import careers1 from '../../public/assets/images/careers1.svg'
+import careers2 from '../../public/assets/images/careers2.svg'
+import pitching1 from '../../public/assets/images/pitching1.svg'
+import pitching2 from '../../public/assets/images/pitching2.svg'
+
+const WorkshopsContainer = styled.div`
+  position: relative;
+  min-height: calc(100vw * (2050 / 1280));
+  top: calc(100vw * (-630 / 1280));
+`
```

Comment:
since this is in the `MiddleSectionsContainer` you can do something like this to place it at the bottom of it
```suggestion
const WorkshopsContainer = styled.div`
  position: absolute;
  bottom: 0;
  width: 100%;
  height: calc(100vw * (550 / 1280));
`
```